tools/sklearn/search_model_validation.xml

```diff
@@ -166,23 +163,16 @@ with open(outfile_estimator, "wb") as output_handler:
                 <param name="search_p" value="k: [-1, 3, 5, 7, 9]"/>
                 <param name="selected_param_type" value="prep_2_p"/>
             </conditional>
-            <param name="error_score" value="false"/>
             <param name="infile1" value="regression_X.tabular" ftype="tabular"/>
             <param name="header1" value="true" />
             <param name="selected_column_selector_option" value="all_columns"/>
             <param name="infile2" value="regression_y.tabular" ftype="tabular"/>
             <param name="header2" value="true" />
             <param name="selected_column_selector_option2" value="all_columns"/>
-            <output name="outfile_result" >
-                <assert_contents>
-                    <has_text_matching expression="[^/d]+0.7938837807353147[^/d]+{u'estimator__C': 1, u'preprocessing_2__k': 9}[^/d]+1" />
-                    <has_text text="0.0"/>
-                </assert_contents>
-            </output>
```

Comment:
What happen to this test? The `error_score` is a very useful feature that was tested here. If regular express can't pass, we may try just plain text.

Reply:
In this test, `<test expect_failure="true">` and `<param name="error_score" value="false"/>` do not go together. This test passes when either `<test expect_failure="true">` is used or `<test expect_failure="true">` and `<param name="error_score" value="true"/>` are used. When we use `<test expect_failure="true">`, it is not important to compare any output. That's why, the output of this test was removed.

https://docs.galaxyproject.org/en/release_18.05/dev/schema.html#id76